pom.xml

```diff
@@ -771,7 +771,7 @@
               </execution>
             </executions>
             <configuration>
-              <source>8</source>
+              <source>17</source>
```

Comment:
we can have at most jdk 11 here as luna 2.10 runs on 11

Reply:
I'll drop that change list